ee/tabby-schema/src/schema/mod.rs

```diff
@@ -564,6 +568,34 @@ impl Query {
         )
         .await
     }
+
+    async fn custom_web_documents(
+        ctx: &Context,
+        after: Option<String>,
+        before: Option<String>,
+        first: Option<i32>,
+        last: Option<i32>,
+    ) -> Result<Connection<WebDocument>> {
```

Comment:
```suggestion
    ) -> Result<Connection<CustomWebDocument>> {
```

Reply:
CustomWebDocument / PresetWebDocument have different field per design doc, any reason to revert it?

Reply:
No. It is just a mistake. I will change it as design document